### DISCUSSION TITLE
Need to filter ELB for which enhanced health monitoring is not enabled.

### DISCUSSION THREAD

**Discussion by dharaghodasara** [cloud-custodian/cloud-custodian#7316] [General]
I want to send notification of those ELB for which enhanced health monitoring is not enabled.

**Comment by thisisshi** [comment]
I'm assuming this is for Elastic Beanstalk, in which case we would need to augment the resource to add  `describe_environment_health`

**Comment by thisisshi** [comment]
I'm assuming this is for Elastic Beanstalk, in which case we would need to augment the resource to add  `describe_environment_health`
